nomos-services/cryptarchia-consensus/src/network/mod.rs

```diff
@@ -18,7 +18,7 @@ type BoxedStream<T> = Box<dyn Stream<Item = T> + Send + Sync + Unpin>;
 #[async_trait::async_trait]
 pub trait NetworkAdapter {
     type Backend: NetworkBackend + 'static;
-    type Settings: Clone + 'static;
+    type Settings: Clone + Send + 'static;
```

Comment:
Maybe here as well, although I'm not so sure in this one.

Reply:
I also wasn't really convinced of adding `Send` to all those `Settings` but let me elaborate on the situation:
1. This is the definition of `RecoveryOperator` for `CryptarchiaConsensus`: `RecoveryOperator<JsonFileBackend<Self::State, Self::Settings>>`
2. The `Backend` needs to be Send in order to use it inside the `StateOperator::run`, which is why it requires `Settings` to be `Send`.

Is there a better way to overcome this?

Reply:
I think you can then bound it for cryptarchia consensus as cryptarchia consensus need it rather than the trait :thinking: 
Can you give it a shot?